ui/app/src/utils/i18n-legacy.ts

```diff
@@ -718,12 +718,15 @@ const adminConfigurationMessages = defineMessages({
   },
   encryptHint: {
     id: 'adminConfigurations.encryptHint',
-    defaultMessage: 'By adding the `encrypted=””` attribute to a tag on the config document you mark it for encryption.' +
-      ' By pressing the “Encrypt Marked” button, the system will encrypt the inner text of each of the marked tags. ' +
-      'Use the `encrypted=””` attribute only on tags which directly hold the value to be encrypted (text). ' +
-      'Don’t add the attribute on tags that contain other tags - unless you actually want to encrypted a chunk of XML. ' +
-      'The encrypted text result will be in the form of `${enc:..}`. This wrapping is a hint to the system that this is ' +
-      'an encrypted value. Please do not manipulate encryption results manually.'
+    defaultMessage: 'To encrypt the content of a tag, (1) mark the desired tags for encryption, then (2) click on the "Encrypt Marked" button. </br></br>' +
+      '(1) Mark your tag/s for encryption by adding the attribute `encrypted=""`. </br>' +
+      'Example: &lt;accessKey encrypted="">AKIAIOSFODNN7EXAMPLE&lt;/accessKey> </br></br>' +
+      '(2) Click on the `Encrypt Marked` button, and your tag should now look like: </br>' +
+      '&lt;accessKey encrypted="true">${enc:xeJW23SomeEncryptedValuesListedHere&lt;/accessKey> </br></br>' +
+      'Remember: </br>' +
+      '* Use the `encrypted=””` attribute only on tags which directly hold the value to be encrypted (text). *' +
+      'Don’t add the attribute on tags that contain other tags - unless you actually want to encrypt a chunk of XML. </br>' +
+      '* Do not manipulate encryption results manually.'
```

Comment:
You shouldn't use HTML tags on the translation. Use XML-like and replace them as formats. Also, make sure you test the output. For bullets, use • (option + 8).